tests/test_devices/test_create_device.py

```diff
@@ -23,16 +22,6 @@ def test_device_creation(self):
         query = self.app_test.post(devices_query, headers=headers)
         self.assertEqual(query.data, devices_query_response)
 
-    def test_create_device_in_room_that_is_not_in_admin_location(self):
```

Comment:
Could you kindly explain why you are removing these test cases?

Reply:
This test case isn't needed again since we're getting the location from admin's token. There used to be a function that returns an error if someone tries to create a device outside his country, I took the function out because we aren't accepting the `location` parameter when creating devices anymore

Reply:
@Bonifase